keps/sig-node/753-sidecar-containers/README.md

```diff
@@ -938,10 +1116,37 @@ Update OOM score adjustment:
 
 ##### Integration tests
 
-The feature will be covered with unit tests and e2e tests.
+<!--
+Integration tests are contained in k8s.io/kubernetes/test/integration.
+Integration tests allow control of the configuration parameters used to start the binaries under test.
+This is different from e2e tests which do not allow configuration of parameters.
+Doing this allows testing non-default options and multiple different and potentially conflicting command line options.
+-->
+
+<!--
+This question should be filled when targeting a release.
+For Alpha, describe what tests will be added to ensure proper quality of the enhancement.
+
+For Beta and GA, add links to added tests together with links to k8s-triage for those tests:
+https://storage.googleapis.com/k8s-triage/index.html
+-->
+
+- <test>: <link to test coverage> FIXME to fill
```

Comment:
@tzneal @gjkim42 can you help me here?

Reply:
Maybe...

`No integration tests are planned. We'll cover this with e2e_node tests.`